packages/rxjs/README.md

```diff
@@ -0,0 +1,67 @@
+# 🔥 @vueuse/rxjs
+
+[![NPM version](https://img.shields.io/npm/v/@vueuse/firebase?color=a1b858)](https://www.npmjs.com/package/@vueuse/rxjs)
+
+> This is an add-on of [VueUse](https://github.com/antfu), enables of a natural way the use of rxjs.
+
+[**Full Documents**](https://vueuse.js.org/)
+
+## 📦 Install
+
+### for Vue 3 ([vue-next](https://github.com/vuejs/vue-next))
+
+<pre class='language-bash'>
+npm i @vueuse/rxjs<b>@vue3</b> rxjs 
+</pre>
+
+### for Vue 2 with [composition-api](https://github.com/vuejs/composition-api) polyfill
+
+<pre class='language-bash'>
+npm i @vueuse/rxjs<b>@vue2</b> rxjs
+</pre>
+
+## ⚡ Functions
+
+`@vueuse/rxjs` provides following functions
+
+<!--GENEARTED LIST, DO NOT MODIFY MANUALLY-->
+<!--FUNCTIONS_LIST_STARTS-->
+
+- Rxjs
+  - [`useObservable`](https://vueuse.js.org/?path=/story/add-ons-rxjs--useobservable) — use an Observable
+  - [`useSubscription`](https://vueuse.js.org/?path=/story/add-ons-rxjs--usesubscription) — uses subscritions without worry about unsubscribe to it or memory leaks
```

Comment:
btw @antfu  here it doesn't taking into consideration the functions "from" and "fromEvent" i used those name because remain the same usage as the originals but allowing vue ref objects

Reply:
You can change the following lines to `f => !f.startsWith('_')`

https://github.com/antfu/vueuse/blob/ef88e9ef003cbcde2b15dae2c44eac8ce9d45b59/scripts/import.js#L13
https://github.com/antfu/vueuse/blob/ef88e9ef003cbcde2b15dae2c44eac8ce9d45b59/scripts/updateReadme.js#L25